common/src/main/java/org/jboss/hal/testsuite/util/Console.java

```diff
@@ -287,6 +287,16 @@ public void pageDown() {
         return this;
     }
 
+    public <T extends BasePage> Console waitForFirstNavigationPanel(Class<T> clazz) {
```

Comment:
Could this method call `refreshAndNavigate` and then wait? To avoid code duplication and increase readability.